docs/UserGuide.adoc

```diff
@@ -157,6 +157,35 @@ There is no need to save manually.
 _{explain how the user can enable/disable data encryption}_
 // end::dataencryption[]
 
+=== Importing a list of orders : `import`
+
+You will be able to import orders via .csv files that are given to you by your company.
+The CSV file will include all relevant and important details of the parcels; such as the customers’ names,
+transaction id of the parcels and the address of the customers, etc. +
+
+Format: `import FILE NAME`
+
+****
+* Import the specific csv with the file name in data folder to the application
+* The FILE NAME should include the extension. For example: *orders.csv*.
+* The CSV file should store at data folder that is at the same directory as the JAR file.
+****
+
+Examples:
+
+* `import orders.csv` +
+Import the contents of the csv file `orders.csv` to Delino.
+
+=== Listing orders : `list`
+
+[None]
+==== Description
+[none]
+* You can request to list the delivery order to see.
+The done status of the delivery order will be used to list the delivery orders which have the same status.
+There are three types of list commands that are shown in the examples below.
+
+
```

Comment:
Might be this additional lines failing the checkstyle test

Reply:
Not that problem. The problem lies with there are no newline at the end of the file. Problem solved by adding newline at the end.